src/controllers/update/update.review.beneficial.owner.individual.ts

```diff
@@ -0,0 +1,34 @@
+import { UPDATE_BENEFICIAL_OWNER_BO_MO_REVIEW_URL, UPDATE_BENEFICIAL_OWNER_TYPE_URL, UPDATE_REVIEW_BENEFICIAL_OWNER_INDIVIDUAL_PAGE } from "../../config";
+import { NextFunction, Request, Response } from "express";
+import { getApplicationData } from "../../utils/application.data";
+import { logger } from "../../utils/logger";
+
+export const get = (req: Request, res: Response) => {
+  logger.debugRequest(req, `${req.method} ${req.route.path}`);
+  const appData = getApplicationData(req.session);
+  const index = req.query.index;
+
+  let dataToReview = {};
```

Comment:
Don't really need to check if we have the list. Better to crash out than show an empty review page.

Reply:
Yes not really needed but since appData.beneficial_owners_individual is of type "BeneficialOwnerIndividual || undefined", the undefined part thus require checking it is not undefined, otherwise compilation error.

Open to suggestions if there is a different way to handle the undefined